src/foam/u2/wizard/StepWizardController.js

```diff
@@ -438,13 +440,11 @@ foam.CLASS({
     },
     function onWizardletCompleted(wizardlet) {
       try {
-        wizardlet.pubAnalyticEvt && this.analyticsAgent?.pub('event', {
-          name: foam.String.constantize(
-              wizardlet.title || wizardlet.id ||
-              (wizardlet.of?.name ?? 'UNKNOWN')
-            ) + '_COMPLETE',
-          tags: ['wizard']
-        })
+        if ( ! this.handleEvent ) return;
+        this.handleEvent(this.WizardEvent.create({
```

Comment:
Wouldn't we need a compounded eventHandler too? So that we can push to iframes and analyticEventDAO?

Reply:
Yeah, that can be added on the nanopay side - handling wizard events with window.postMessage seems a bit specific for the package in foam